modules/ROOT/pages/penetration-testing-policies.adoc

```diff
@@ -17,15 +17,18 @@ against Anypoint Platform, request a Security Assessment before proceeding.
 
 To request a Security Assessment:
 
-. Contact your MuleSoft Customer Success Manager to request the Security Assessment Request form and Security Assessment Agreement.
+. Navigate to: https://security.secure.force.com/securityassessments and click on *Sign-in with SecureAssess* if you don't have a Salesforce account.
+. Select *New User* and register your details. Once registered and approved, you'll be able to log-in to the portal and submit your pen test request.
```

Comment:
Per style ref:
```suggestion
. Select *New User* and register your details. After you are registered and approved, you can log in to the portal and submit your pen test request.
```

Reply:
However...at this point I read further into the topic, and I sense that the information is not chronological. I wonder if you would prefer to organize this way (I shortened for convenience; feel free to add to it):

1. Navigate to...
2. If you do not already have a Salesforce account, click...
3. Select *New User*.
4. Complete the request form [[optional] with the details you want to register.] See <link>.
5. Accept the agreement and wait for automated approval.
6. If you do not receive the approval notice, check your Spam folder before contacting Salesforce.

After you are approved, you can log in to the portal and submit your pen test request.

NOTE: If you prefer, you can request approval manually by submitting the Security Assessment Request to <email>.